components/Observations.tsx

```diff
@@ -53,62 +58,130 @@ export const Observations: React.FunctionComponent<{
     />
   );
 };
 
+export const zone = (mapLayer: MapLayer, lat: number, long: number): string => {
+  const matchingFeatures = mapLayer.features.filter(feature => geoContains(feature.geometry, [long, lat])).map(feature => feature.properties.name);
+  if (matchingFeatures.length === 0) {
+    return 'Unknown';
+  } else if (matchingFeatures.length > 1) {
+    // TODO: this happens almost 100% ... why?
+    // also, seems like the widget is naming things with more specificity than just the forecast zones? e.g. teton village
+    console.log(`(${long},${lat}) matched ${matchingFeatures.length} features: ${matchingFeatures}`);
+  }
+  return matchingFeatures[0];
+};
+
 export const ObservationSummaryCard: React.FunctionComponent<{
   observation: OverviewFragment;
   zone: string;
 }> = ({zone, observation}) => {
   const navigation = useNavigation<ObservationsStackNavigationProps>();
-  const {width} = useWindowDimensions();
+  const anySignsOfInstability =
+    observation.instability.avalanches_caught ||
+    observation.instability.avalanches_observed ||
+    observation.instability.avalanches_triggered ||
+    observation.instability.collapsing ||
+    observation.instability.cracking;
   return (
     <Card
-      px="2"
-      pt="2"
+      marginTop={2}
+      borderRadius={8}
+      borderColor="white"
       onPress={() => {
         navigation.navigate('observation', {
           id: observation.id,
         });
       }}
       header={
-        <HStack justifyContent="space-between" alignItems="center">
-          <Heading size="md" flex={1} mr="8" fontWeight="normal">
-            {zone + ' - ' + observation.locationName}
-          </Heading>
-          <Heading size="md" color="light.400" fontWeight="normal">
-            {observation.startDate}
-          </Heading>
-        </HStack>
+        <Row alignContent="flex-start" flexWrap="wrap" alignItems="center" space="2">
+          <Title3Semibold>{zone}</Title3Semibold>
+          <FontAwesome name="angle-right" size={18} style={{paddingHorizontal: 4}} color="black" />
+          <Title3Semibold>{observation.locationName}</Title3Semibold>
+        </Row>
       }>
-      <RenderHTML source={{html: observation.observationSummary}} contentWidth={width} />
-      <HStack justifyContent="space-between" pt="4">
-        <NBText color="light.400">{observation.name}</NBText>
-        <NBText color="light.400">{observation.observerType}</NBText>
-      </HStack>
+      <Row flexWrap="wrap" space="2">
+        <IdentifiedInformation header={'Submitted'} body={dateToString(observation.createdAt)} />
+        <IdentifiedInformation header={'Observer'} body={FormatPartnerType(observation.observerType as PartnerType)} />
+        <IdentifiedInformation header={'Author(s)'} body={observation.name || 'Unknown'} />
+      </Row>
+      {anySignsOfInstability && (
+        <Column space="2" style={{flex: 1}}>
+          <Text bold style={{textTransform: 'uppercase'}}>
```

Comment:
maybe use `Body` instead of `Text` (fine if you want to handle in a follow-up PR)

Reply:
This is what I wanted your help on :)